client/components/list-end/README.md

```diff
@@ -1,7 +1,7 @@
 ListEnd (JSX)
 ====================
 
-This component is used to display a WordPress logo in the centre of a line, usually used at the end of a list.
+This component is used to display a WordPress logo in the centre of a line. It should be used to mark the end of content in an [InfiniteList](https://github.com/Automattic/wp-calypso/blob/master/client/components/infinite-list/README.md).
```

Comment:
We must use a relative path here instead of linking directly to GitHub's version of the file.

Reply:
I've updated it, but unless I'm missing something, I think the problem with that is then the link in DevDocs is broken: `/devdocs/design/list-end`

<img width="1117" alt="Screenshot 2020-04-02 at 08 31 53" src="https://user-images.githubusercontent.com/43215253/78222091-7f802680-74bc-11ea-8d92-4f2c2598c71e.png">

Any idea if there's a way to get it working there too? :)

Reply:
Completely relative should work since they're in the same folder: `[InfiniteList](./infinite-list)`. Though we don't have an example page for infinite list so the following will render:

<img width="1484" alt="Screen Shot 2020-04-02 at 3 28 01 PM" src="https://user-images.githubusercontent.com/1270189/78305836-c854d100-74f6-11ea-8aff-71ac799eba90.png">

Reply:
Not sure why the 404, @Aurorum, let me research on that later today. Let us know if you update the link with Kerry's suggestion though.

Reply:
Thanks! Won't that cause an issue with GitHub though? It's leading me to this: https://github.com/Aurorum/a-calypso-forked/blob/fix/listend-readme/client/components/list-end/infinite-list

Is there no way of getting them both to work? :)